### PR TITLE
Skip invoice paid twice error in NWC

### PR DESCRIPTION
**Pull request by benthecarman**
We handle this for auto approvals in the backend but can't for manual. If we've already paid an invoice no real reason to show an error, they've already paid it so only way forward is to skip and go on so this just changes to skip those errors

### CODE REVIEW

**Review by TonyGiorgio — APPROVED** [review]
concept ack

File: src/components/PendingNwc.tsx [hunk 1]
```diff
@@ -78,8 +78,19 @@ export function PendingNwc() {
             await state.mutiny_wallet?.approve_invoice(item.id);
             await vibrateSuccess();
         } catch (e) {
-            setError(eify(e));
-            console.error(e);
+            // If we've already paid this invoice, just ignore the error
+            // we just want to remove it from the list and continue
+            if (e === "An invoice must not get payed twice.") {
```
Comment (futurepaul) [inline src/components/PendingNwc.tsx:83]: it's usually `e.message`, not the `e` object itself. is there a way we can test this?

**Review by futurepaul — CHANGES_REQUESTED** [review]

File: src/components/PendingNwc.tsx (hunk 1 above)
Comment (benthecarman) [reply src/components/PendingNwc.tsx]: hard to test, this is a pretty uncommon edge case

**Review by futurepaul — APPROVED** [review]
